pkg/okteto/context.go

```diff
@@ -406,6 +407,9 @@ func encodeOktetoKubeconfig(cfg *clientcmdapi.Config) string {
 	for name, cluster := range cfg.Clusters {
 		if name == currentCluster {
 			cluster.LocationOfOrigin = ""
+			if len(cluster.Extensions) > 0 {
+				cluster.Extensions = make(map[string]runtime.Object)
```

Comment:
is it possible to keep them? they might be needed to access the cluster. for example, telepresence adds an extension to specify on which namespace the telepresence server is running